doc/dftb+/manual/dftbp.tex

```diff
@@ -2503,7 +2507,7 @@ \section{Options}
   \kw{WriteDetailedOut} & l & & Yes & \\
   \kwl{RestartFrequency}{kw:dftbp.RestartFrequency} & i &Driver = \cb, SCC = Yes & 20 & \\
   \kw{RandomSeed} & i & & 0 & \\
-  \kw{MinimiseMemoryUsage} & l & & No & \\
```

Comment:
See comment below on depreciation

Reply:
Added back, mentioning that it has no effect. :smile:  When we implement on-the-fly eigenvector recalculation before the release, we can rewrite it accordingly. Otherwise we should marked as removed, bump parser version and add it to oldcompat...